inst/pages/add-comm-typing.Rmd

```diff
@@ -330,7 +330,7 @@ And now nMDS.
 ```{r message = FALSE, warning = FALSE}
 tse2  <- runNMDS(tse2, FUN = vegan::vegdist, method = "bray", 
                 name = "NMDS_BC", assay.type = "counts", ncomponents = 20)
-scater::plotReducedDim(tse2, "NMDS_BC", colour_by = "CST", point_alpha = 1) + th + 
+scater::plotReducedDim(tse2, "NMDS_BC", colour.by = "CST", point.alpha = 1) + th + 
```

Comment:
scater::plotReducedDim still have the paramer names colour_by... They have not changed them, check